tests/infra/consortium.py

```diff
@@ -302,9 +302,14 @@ def get_proposals(self, remote_node):
 
     def retire_node(self, remote_node, node_to_retire):
         LOG.info(f"Retiring node {node_to_retire.local_node_id}")
-        proposal_body, careful_vote = self.make_proposal(
-            "retire_node", node_to_retire.node_id
-        )
+        if os.getenv("JS_GOVERNANCE"):
```

Comment:
We might as well rename the hard-coded C++ proposal too, so that the user experiencer is somehow smoother if we keep both Lua and JS governance together in the next release?

Reply:
I disagree, the point of keeping the old governance around is that it hasn't changed and still works. Changes are only necessary when changing to JS.